### DISCUSSION TITLE
Edit Readme to include section about nvm to ensure proper node version when working localy

### DISCUSSION THREAD

**Discussion by tmsteph** [DustinBrett/daedalOS#257] [Feature Requests]
I had trouble running daedalOS locally. 

It turns out my local node version was 18 which was causing it not to load. 

I had to do research to find out why. 

Node Version Manager aka nvm was a simple way to easily downgrade. 

Should I suggest an edit to the readme?

Also there is a way to have a better error message for this problem. 

add 

"engines": {
  "node": "16"
}

to the packages.json file. 

should I submit this as an issue or pull request?

**Comment by DustinBrett** [comment]
Thanks for the suggestion of adding `engines`. I will do that. As for mentioning it in the README, I actually do say Node v16 LTS, but that isn't LTS anymore so I will update the docs and make it more clear.

**Reply by DustinBrett** [reply]
Also if 18 is LTS now then I'd like to get it working with that ideally.

**Comment by DustinBrett** [comment]
I've got it working with Node 18 in Docker/Windows.

https://github.com/DustinBrett/daedalOS/commit/3a59152369ea40e591a39b21d32c32a9be9aa93d
